src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC, Dispatch, SetStateAction } from 'react';
+import classNames from 'classnames';
+import { capitalizeString } from '../../utils/capitalizeString';
+import { StatusFilter } from '../../types/StatusFilter';
+
+type Props = {
+  filterValue: StatusFilter;
+  setFilterValue: Dispatch<SetStateAction<StatusFilter>>;
```

Comment:
u can use another syntax(up to u)

```suggestion
  setFilterValue: (filterValue: StatusFilter) => void;
```